[EdgeTPU] Add EdgeTPUCompiler parseVersion testcode

profornnan opened the pull request:
- Implemented code to test the parseVersion() method
- Formatting EdgeTPUToolchain.test.ts

ONE-vscode-DCO-1.0-Signed-off-by: profornnan <profornnan@naver.com>

profornnan committed:
[EdgeTPU] Add EdgeTPUCompiler parseVersion testcode

- Implemented code to test the parseVersion() method

ONE-vscode-DCO-1.0-Signed-off-by: profornnan <profornnan@naver.com>

profornnan committed:
[EdgeTPU] Style EdgeTPUToolchain

- Formatting EdgeTPUToolchain.test.ts

ONE-vscode-DCO-1.0-Signed-off-by: profornnan <profornnan@naver.com>

rhqjatn2398 commented on src/Tests/Backend/EdgeTPU/EdgeTPUToolchain.test.ts:
```diff
@@ -132,3 +135,44 @@ suite("Backend", function () {
+      const version2 = new Version(2, 1, 302470888);
+      assert.deepEqual(version1, version2);
+    });
+    test("returns Version object from string version without minor", function () {
```
Umm... I think we should change the test title to "returns Version object from string version without patch".

profornnan replied:
As you suggested, it seems there are no cases where only major and patch are used without minor, so I have updated the title. Thank you!!

profornnan committed:
[EdgeTPU] Fix Toolchain parseVersion testcode title

- string version without minor => string version without patch

ONE-vscode-DCO-1.0-Signed-off-by: profornnan <profornnan@naver.com>

rhqjatn2398 reviewed (approved):
LGTM 👍
